frontend/src/mlmd/Utils.tsx

```diff
@@ -117,3 +117,7 @@ export function getMetadataValue(value?: Value): string | number | Struct | unde
       return '';
   }
 }
+
+export function getMetadataValueV2(value?: Value): Struct | undefined {
```

Comment:
We used `getMetadataValue` as a function name previously to accept different type of input. For the current `execution`, the value of `inputs` and `outputs` is guaranteed to be a `Struct` type. I think using `getMetadataValue` is better to tracking to changes in different version; however, `getStructValue` is better to understand the functionality.

Reply:
1. Looking at the `getMetadataValueV2()`: I think it doesn't handle the case where value is not struct type. Hence I think the naming should be more specific. Also, the function itself is not V2 specific, it applies to any proto value.
2. Looking at the `getMetadataValue()`: Can we reuse this method in your PR? In that case, you don't need to define a new function. If not, I am curious what is the difference that causes it unable to reuse.

Reply:
Agree with your first statement!
I chose to use another function is to handle the `Struct` type., because the typeof check doesn't contain `Struct` type (I forgot it's the same as `object` type). It totally makes sense to just reuse the original function even though we guarantee the value is `Struct` type!